makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
+		debug(D_MAKEFLOW_RUN, "setting dag_node->umbrella_spec...\n");
+		struct dag_node *cur;
+		cur = d->nodes;
+		while(cur) {
+			dag_node_set_umbrella_spec(cur, wrapper_umbrella->spec);
```

Comment:
If we are ever setting for each specific node then there will be no need to loop over them all and set the same one (we could still use the generic one in the wrapper). This is duplicate information and it isn't used in the current implementation.